projects/cardiovascular-patient-app/src/styles.scss

```diff
@@ -33,4 +33,38 @@ body {
 .red-snackbar {
   --mdc-snackbar-container-color: red;
   --mat-mdc-snack-bar-button-color: #e8e9ec;
+}
+
+// This will make table fixed with so if text increases it will wrap onto next line and will not add horizotal scrollbar
+
+#summary-card-table {
+  .mat-mdc-table {
+    table-layout: fixed;
+    white-space: normal;
+    .mat-mdc-cell{
+        white-space: normal;
+    }
+  }
+}
+
+// This will minus the header height from the content to remove unnecessary scrollbar 
+
+.bp-tab-group {
+  height: calc(100% - 60px);
+  .mat-mdc-tab-body-wrapper {
+    height: 100%;
+  }
+}
+
+.report-bp-card .mat-mdc-card {
+  box-shadow: none;
+}
+
+.bp-datepicker {
+  min-width: 300px;
+}
+
+.bp-datepicker .mat-mdc-form-field-infix {
+  max-width: 180px;
+  width: 100%;
```

Comment:
why are you using global styles for these? can they be added to the component stylesheet instead?

Reply:
@semicolin in this CSS some classes generate dynamically at run time, if we use these classes for CSS in the component level we are required to use::ng-deep  for applying CSS for dynamic CSS, but it is deprecated, without::ng-deep the CSS is not applied for dynamic class at the component level. so we used it without the ng-deep in global CSS.